backend/backend/AimTrainerGame/Controllers/AimTrainerGameController.cs

```diff
@@ -27,4 +30,32 @@ public ActionResult<GameStartResponse> StartGame([FromBody] GameStartRequest gam
 
         return Ok(new GameStartResponse(dots, difficultySettings.dotCount, difficultySettings.timeToLive));
     }
+
+    [HttpGet("Highscores")]
+    public ActionResult<IEnumerable<Highscore>> GetHighscores([FromQuery] int amount = 10)
+    {
+        Console.WriteLine(amount);
+        var highscores = _db.AimTrainerGameHighscores
+            .OrderByDescending(h => h.Score)
+            .ThenBy(h => h.Date)
+            .Take(amount);
+
+        return Ok(highscores);
+    }
+
+    [HttpPost("Highscores")]
+    public ActionResult<Highscore> EndGame([FromBody] GameEndRequest gameInfo)
```

Comment:
Not completely sure, bet ar cia reikia grazint ActionResult<Highscore>? Nes kiek ziurejau, man atrodo cia tiesiog uztenka grazint ActionResult<void> ar kazka panasaus, kad to Highscore nereiktu.

Reply:
Nelabai supratau ka omenyje turi

Reply:
Tipo something like ActionResult. Nes nzn ar reikia ten grazint ActionResult<Highscore>.

Reply:
Galimai specifiskai nezymeti tipo, bet manau geriau pazymeti, ir aiskiau, ir padeda galimai swaggeriui, not sure